README.md

```diff
@@ -139,17 +139,21 @@ You'll need a C3 or C3X and a Toyota A harness for current supported vehicles.
 
 ### Step 1. Power up C3X, connect to Wi-Fi, and install `commaai/master-ci`
 
-1-1. Start with the official installation guide: https://comma.ai/setup/comma-3x
+1-1. Install the hardware by following the official Installation Guide: https://comma.ai/setup/comma-3x
 
-1-2. Always use the right-angled USB-C cable that came with the Comma 3X. Don't use your own USB cable. A USB-C 3.1 Gen 2 is required.
+1-2. Turn on your Comma 3X by plugging in the right-angled OBD-C cable to the harness.
 
-1-3. Also connect the OBD2 connector to make sure the Comma stays powered on while turning the car on and off.
+> [!WARNING]
+> * For connecting Comma to the car harness, always use the right-angled OBD-C cable that came with the Comma 3X. The hardness sends 12V instead of the usual 5V, so your own USB cable will not work.
+> * Comma AI sells it if you need more: https://comma.ai/shop/obd-c-cable
+> * If you must buy your own, USB-C 3.1 Gen 2 is required.  
+> * For turning on Comma at home, your phone charger works fine. It will show a voltage warning but it will work.
```

Comment:
This should be noted as a YMMV. The C3 doesn't really follow standards and so far, the most success has been USB A-C but C-C sometimes works too.